cypress/integration/Components/Autocomplete.feature

```diff
@@ -90,7 +90,7 @@ Feature: As a user, I want to be able to use autocomplete features to it's full
 
     Scenario: as a user, I should be able to touch screen to select item
         Given the user navigates to "/about-cancer/treatment/clinical-trials/search"
-        When user types "bre" in "Cancer Type/Keyword" field
+        When user types "breast cancer" in "Cancer Type/Keyword" field
```

Comment:
I'm sure you know I am going to ask why this was changed ?)

Reply:
..saves me from asking

Reply:
@kate-mashkina @arcepaul ^^^